server/src/test/java/io/crate/integrationtests/AlterTableIntegrationTest.java

```diff
@@ -230,6 +230,25 @@ public void test_alter_partitioned_table_drop_column_can_add_again() {
     private void execute_statements_that_drop_and_add_column_with_the_same_name_again(boolean partitioned) {
         // Re-add top-level column with same name and different type.
         execute("alter table t drop column a");
+
+        // Exclude partitioned by column in order not adjust assertions for regular/partitioned tables.
+        execute("""
```

Comment:
Please don't add even more test cases to this test, use a dedicated `PgCatalogITest` instead.

Reply:
Thanks, while creating a dedicates test, spot an issue - will create a separate PR since it's related to assigning column positions and not to this change.